kafka/tests/common.py

```diff
@@ -0,0 +1,93 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import socket
+
+from datadog_checks.dev import get_docker_hostname, get_here
+
+CHECK_NAME = "kafka"
+
+HERE = get_here()
+HOST = get_docker_hostname()
+HOST_IP = socket.gethostbyname(HOST)
+KAFKA_CONNECT_STR = '{}:9092'.format(HOST_IP)
+ZK_CONNECT_STR = '{}:2181'.format(HOST)
+TOPICS = ['marvel', 'dc']
+PARTITIONS = [0, 1]
+
+# Rate type metrics that do not work in our e2e:
+
+# "kafka.request.fetch.failed.rate",
+# "kafka.request.fetch.failed_per_second",
+# "kafka.request.fetch.time.99percentile",
+# "kafka.request.fetch.time.avg",
+
+# "kafka.request.produce.failed.rate",
+# "kafka.request.produce.failed_per_second",
+
+#   "kafka.messages_in.rate"
```

Comment:
nit- could you make this a block comment and also make the spaces consistent?

Reply:
I updated this and used the block comment format